dev/spark_overview.md

```diff
@@ -0,0 +1,452 @@
+# Apache Spark Overview
+This page describes Spark concepts that are important to understand when using dsgrid.
+
+[Cluster Mode Overview](#cluster-mode-overview) |
+[Run pyspark through iPython or Jupyter](#run-pyspark-through-ipython-or-jupyter) |
+[Creating a SparkSession with dsgrid](#creating-a-sparksession-with-dsgrid) |
+[Installing a Spark Standalone Cluster on Your Laptop](#installing-a-spark-standalong-cluster-on-your-laptop) |
+[Installing a Spark Standalone Cluster on an HPC](#installing-a-spark-standalong-cluster-on-an-hpc) |
```

Comment:
I tried to add a relative link to the EMR readme, but it didn't work. If someone knows if that can be done with a parent directory, let me know. Otherwise, I can add a hard link to the main page.